metricbeat/docs/fields.asciidoc

```diff
@@ -3463,6 +3463,40 @@ type: long
 
 
 
+[float]
+== com Fields
```

Comment:
Not sure about "com". Perhaps we could name it "commands" or "queries"? @ruflin, what do you think?